docs/source-fabric/api/fabric_methods.rst

```diff
@@ -61,6 +61,41 @@ This replaces any occurrences of ``loss.backward()`` and makes your code acceler
     fabric.backward(loss)
 
 
+clip_gradients
```

Comment:
Why don't we use Sphinx's automethod directive and write everything inside the method docstring?

Reply:
Because the docstrings are meant for reference, including types etc., while the written docs in the .rst are for user guide. They can be longer and have more examples etc. They may also link to other docs, include images, code blocks etc. In general, one should not mix user guide with docstrings.